### PR TITLE
Setup openshift/cluster-bootstrap

### PR DESCRIPTION
**Pull request by tnozicka**
/cc @sttts

### CODE REVIEW

**Comment by tnozicka** [comment]
(just added @mfojtik as well)

File: ci-operator/config/openshift/cluster-bootstrap/openshift-cluster-bootstrap-master.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+images:
+  - dockerfile_path: Dockerfile
+    from: base
+    to: cluster-bootstrap
+
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+  tag: ''
```
Comment (droslean) [inline ci-operator/config/openshift/cluster-bootstrap/openshift-cluster-bootstrap-master.yaml]: You can remove this

File: ci-operator/config/openshift/cluster-bootstrap/openshift-cluster-bootstrap-master.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  tag_overrides: {}
```
Comment (droslean) [inline ci-operator/config/openshift/cluster-bootstrap/openshift-cluster-bootstrap-master.yaml]: this too

**Comment by tnozicka** [comment]
@droslean updated based on your comments

**Comment by tnozicka** [comment]
@smarterclayton would you please setup publishing for this new image?

**Pull request merged**